compiler/quilt/tools/data_transfer.py

```diff
@@ -64,6 +64,13 @@ def download_fragments(store, obj_urls, obj_sizes):
     # Some objects might be missing a size; ignore those for now.
     total_bytes = sum(size or 0 for size in itervalues(obj_sizes))
 
+    # Check if we have enough disk space. There's no way to check reliably because we also need
+    # space for the temporary gzip'ed files, but that's better than nothing.
+    free_space = 5 # get_free_space(store.object_path('.'))
```

Comment:
i don't understand why `get_free_space` is commented out.

Reply:
does this respect `QUILT_PRIMARY_PACKAGE_DIR`?

Reply:
Oh crap, was testing out of space errors. The fact that it didn't break any unit tests is kinda bad...

Yeah, it checks the directory where the objects will actually get saved,